lib/tasks/export.rake

```diff
@@ -5,7 +5,10 @@ namespace :export do
 
   desc 'Export task entities to CSV'
   task :tasks, [:output] => [:environment] do |_task, args|
-    Export::Anything.new(Task.all, args[:output]).run
+    Export::Anything.new(
+      Task.select('tasks.*, frameworks.short_name').joins(:framework).includes(:framework),
```

Comment:
is the select an optimisation to avoid loading all the framework fields? perhaps a tad over-zelous?

Reply:
Perhaps 😄 

That's just where my head's at rn.